kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -300,7 +297,11 @@ private synchronized void manageConfiguredInterfaces(List<String> configuredInte
                 continue;
             }
 
-            manageConfiguredInterface(iface, properties);
+            try {
+                manageConfiguredInterface(iface, properties);
+            } catch (Exception e) {
+                logger.error("Unable to configure iface {}, skipping", iface, e);
+            }
```

Comment:
This is correct but we need to do the same for `manageNonConfiguredInterfaces`. I think you can just mirror the logic you have here there.